src/BenchmarkDotNet/Exporters/MarkdownExporter.cs

```diff
@@ -170,13 +173,13 @@ private void PrintTable(SummaryTable table, ILogger logger)
             var separatorLine = Enumerable.Range(0, table.ColumnCount).Select(_ => "").ToArray();
             foreach (var line in table.FullContent)
             {
-                if (rowCounter > 0 && table.FullContentStartOfLogicalGroup[rowCounter] && table.SeparateLogicalGroups)
+               if (rowCounter > 0 && table.FullContentStartOfLogicalGroup[rowCounter] && table.SeparateLogicalGroups)
```

Comment:
nit: we need that space back to keep the code aligned :)

```suggestion
                if (rowCounter > 0 && table.FullContentStartOfLogicalGroup[rowCounter] && table.SeparateLogicalGroups)
```